packages/shared/src/components/cards/v1/PlaceholderCard.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { forwardRef, HTMLAttributes, ReactElement, Ref } from 'react';
+import classNames from 'classnames';
+import { Card, CardSpace, CardTextContainer } from './Card';
+import { ElementPlaceholder } from '../../ElementPlaceholder';
+import classed from '../../../lib/classed';
+
+const Text = classed(ElementPlaceholder, 'rounded-xl');
+
+export type PlaceholderCardProps = {
+  showImage?: boolean;
+} & HTMLAttributes<HTMLDivElement>;
+
+export const PlaceholderCard = forwardRef(function PlaceholderCard(
+  { className, showImage, ...props }: PlaceholderCardProps,
+  ref: Ref<HTMLElement>,
+): ReactElement {
+  return (
+    <Card aria-busy className={classNames(className)} {...props} ref={ref}>
+      <CardTextContainer>
+        <CardSpace className="mb-2 flex flex-row">
+          <ElementPlaceholder className="h-10 w-10 rounded-full" />
```

Comment:
For these elements with equal w-h we can actually use `size-10` these days, it just got added to our tailwind version 🎉

Reply:
Nice, will do that 👍